SecurityManager/Clients.Web/res/Themes/NovaViso/Image/sprite.svg

```diff
@@ -37,19 +37,43 @@
-  <image id="HierarchyThisAndParentAndChildren" height="16" width="16" href="data:image/gif;base64,R0lGODlhEAAQAPcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAAP8ALAAAAAAQABAAAAg5AP8JHPgPAACCCAkaNJgQ4cKFDQU+fNhw4kSHFi9G3CgxI8WBHj9yjBgSosKQFTNutDhSJEeGCQMCADs=" />
+  <svg class="sprite-part" id="HierarchyThis" width="16" height="16" viewBox="0 0 16 16">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+      <rect fill="#000000" x="4" y="6" width="8" height="1"/>
+      <rect fill="#000000" x="4" y="9" width="8" height="1"/>
+    </g>
+  </svg>
+  <svg class="sprite-part" id="HierarchyThisAndChildren" width="16" height="16" viewBox="0 0 16 16">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+      <rect fill="#000000" x="4" y="6" width="8" height="1"/>
+      <rect fill="#000000" x="4" y="9" width="8" height="1"/>
+      <polyline stroke="#000000" stroke-width="1.2" stroke-linecap="square" stroke-linejoin="round" transform="translate(8.000000, 13.664795) scale(1, -1) rotate(-90.000000) translate(-8.000000, -13.664795) " points="6.5 16.6647949 9.5 13.6647949 6.5 10.6647949"/>
+    </g>
+  </svg>
+  <svg class="sprite-part" id="HierarchyThisAndParent" width="16" height="16" viewBox="0 0 16 16">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+      <polyline stroke="#000000" stroke-width="1.2" stroke-linecap="square" stroke-linejoin="round" transform="translate(8.000000, 2.335205) scale(-1, 1) rotate(-90.000000) translate(-8.000000, -2.335205) " points="6.5 5.33520508 9.5 2.33520508 6.5 -0.664794922"/>
+      <rect fill="#000000" x="4" y="6" width="8" height="1"/>
```

Comment:
I patched the hierachy icons ((pushed commit) to repair the semantics. the different combinations dropped part of the center line, which is now fixed.